build.rs

```diff
@@ -0,0 +1,53 @@
+fn main() {
+    manage_docs();
+}
+
+#[cfg(any(feature = "embed-lgpl-docs", feature = "purge-lgpl-docs"))]
+fn manage_docs() {
+    extern crate lgpl_docs;
+    const PATH: &str = "src";
+    const IGNORES: &[&str] = &[
+        "lib.rs",
+        "prelude.rs",
+        "signal.rs",
+        "boxed.rs",
+        "byte_array.rs",
+        "bytes.rs",
+        "char.rs",
+        "clone.rs",
+        "enums.rs",
+        "error.rs",
+        "gobject/mod.rs",
+        "log.rs",
+        "main_context.rs",
```

Comment:
This one should be possible to fill in automatically, mostly.

Reply:
I imagine several of these can be cleared up easily, but these were all the ones where purge lost docs for them which needed more consideration than I had brainpower for at the time.

Reply:
Fair enough :)